developer/src/kmc-keyboard-info/src/validate-mit-license.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Returns an error string for license issues, or null if no issues found
+ * @param license
+ * @returns
+ */
+export function validateMITLicense(license: string) {
+  // NOTE: this function returns error messages which will not be easily
+  // localizable. Given this is only for use with the keyboards and
+  // lexical-models repositories, suggest we don't worry about this
+
+  // split input text into paragraphs, trimming whitespace
+  const text = license.split('\n').map(line => line.trim()).join('\n').split(/\n\s*\n/);
+
+  // MIT license, cleanup whitespace
+  const clauses = [
```

Comment:
Is this sourced from https://opensource.org/license/mit/ ? (so we know where to update)
I see that has 
```
Copyright <YEAR> <COPYRIGHT HOLDER>
```

though copyright is tested separately

Reply:
hmm, the source also doesnt' start with
```
The MIT License (MIT)
```

Reply:
The text of the MIT license is widely shared and static, but we could reference the link you give in the comments at the start of this file?

Reply:
See also https://spdx.org/licenses/MIT.html